core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -80,6 +82,11 @@ object ConfigCommand extends Logging {
         System.err.println(e.getMessage)
         Exit.exit(1)
 
+      case e: UnsupportedVersionException =>
+        logger.debug(s"Unsupported API encountered in server when executing config command with args '${args.mkString(" ")}'")
+        e.printStackTrace(System.err)
```

Comment:
nit: It may be better to use `System.err.println(e.getMessage)`. What do you think?

Reply:
Yes, since it's an expected error, it's unnecessary to print the full stack trace.